benchmark/galsim/galsim_field.py

```diff
@@ -11,20 +11,22 @@
 FITS_CATALOG_FILENAME = os.path.join('output', 'galsim_field_500_catalog.fits')
 ARCSEC_PER_DEGREE = 3600.
 FIELD_EXPAND_ARCSEC = 10
+SKY_LEVEL_NMGY = 0.155 # similar to SDSS
+COUNTS_PER_NMGY = 180.0 # similar to SDSS
 
 def set_image_dimensions(test_case, catalog_rows):
     min_ra_deg = min(float(row['right_ascension_deg']) for row in catalog_rows)
     max_ra_deg = max(float(row['right_ascension_deg']) for row in catalog_rows)
     min_dec_deg = min(float(row['declination_deg']) for row in catalog_rows)
     max_dec_deg = max(float(row['declination_deg']) for row in catalog_rows)
 
-    width_arcsec = (max_ra_deg - min_ra_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
-    height_arcsec = (max_dec_deg - min_dec_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
+    height_arcsec = (max_ra_deg - min_ra_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
+    width_arcsec = (max_dec_deg - min_dec_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
```

Comment:
I think pixel coordinates actually begin at (0.5, 0.5). That way the center of the first pixel is (1.0,1.0). Since Julia is 1-based, `[1,1]` also happens to be the index of the first pixel. In C the first pixel's center still has coordinates (1.0,1.0), but its index is `[0,0]`. Tricky stuff.